### PR TITLE
fix: copying layers on Windows OS (2)

### PR DESCRIPTION
**Pull request by userquin**
This PR also includes a protection to avoid copying layers when running `preview` command.

@filrak did you run eslint in your local? `@antfu/eslint-config` not using semi

gh layer not tested on my local

supersedes #1

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,135 +1,156 @@
...
+interface PluginOptions {
+  external?: ExternalLayer
+}
+
+export default function layeredFilesPlugin(options: PluginOptions = {}): AstroIntegration {
+  return {
+    name: 'astro-layered-files',
+    hooks: {
+      'astro:config:setup': async ({ command, config, updateConfig }) => {
+        // we don't want to copy and watch layers when running the preview command
```
Comment (userquin) [inline src/index.ts]: update hint, watcher added only when the command is `run`

**Comment by filrak** [comment]
Thank you!

**Comment by userquin** [comment]
I suggest you to use squash instead merge pr, this way only 1 commit on main branch (check commit history)

**Comment by filrak** [comment]
Good suggestion, thanks! This repo is generally a bit chaotic and not contribution-friendly. It's a side project being part of an attempt to get back to coding after 3 years break.

Now that there is also someone willing to help (which I am very grateful for!) I will play a bit with conventions, pipelines etc.

May I ask if you want to use this lib yourself or you're just contributing for fun? Either way I can grant you admin rights so you can easily make adjustments by yourself if you're interested.

**Pull request merged**